{
  "contest": 368,
  "handle": "jolah1",
  "risk": "2",
  "title": "`executeWithdraw()` can cause denial of service attack.",
  "issueId": 1326,
  "issueUrl": "https://github.com/code-423n4/2024-04-noya-findings/issues/1326"
}